src/main.cpp

```diff
@@ -138,6 +138,14 @@ QCoreApplication* createApplication(int& argc, char* argv[], unsigned int& actio
     }
 }
 
+void checkAndCopyFont(const QString& pathName, const QString& fileName)
```

Comment:
I think it's better to call it `copyFont`. The check doesn't say anything about the kind of check it does and it's obvious a function will do some checks before it does something.

Reply:
OK... will do.